ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/config/TestConfigLoader.java

```diff
@@ -40,7 +40,7 @@ public static SpecConfig loadPhase0Config(
   }
 
   private static URL getLegacyMainnetConfigResourceAsUrl(final String configName) {
-    final String resourcePath = "tech/pegasys/teku/spec/config/legacy/" + configName + ".yaml";
+    final String resourcePath = "tech/pegasys/teku/spec/config/standard/" + configName + ".yaml";
```

Comment:
It's slightly suspect that this method is called `getLegacyMainnet...` but now doesn't get the legacy config. Are the tests that use it still relevant and the method should be renamed or are the tests now irrelevant?

Reply:
ah i fixed the functionality, yeh but I should probably clean that up!